docs/migration-guide.md

```diff
@@ -80,29 +103,235 @@ val MIGRATION_HREF = object : Migration(1, 2) {
 }
 ```
 
-### LcpDialogAuthentication now supports configuration changes.
+### Error management
+
+Most APIs now return an `Error` instance instead of an `Exception` in case of failure, as these objects are not thrown by the toolkit but returned as values.
+
+It is recommended to handle `Error` objects using a `when` statement. However, if you still need an `Exception`, you may wrap an `Error` with `ErrorException`, for example:
+
+```kotlin
+assetRetriever.sniffFormat(...)
+    .getOrElse { throw ErrorException(it) }
+```
+`UserException` is also deprecated. The application now needs to provide localized error messages for toolkit errors.
+
+### Navigator
+
+#### Click on external links
+
+Clicking on external links is no longer managed by the navigator. To open the link yourself, override `HyperlinkNavigator.Listener.onExternalLinkActivated`, for example:
+
```

Comment:
By the EPUB navigator? I'm not so sure about PDF.

Reply:
Yes, with Pdfium it's not supported for sure. PSPDFKit handles external links, I'm not sure we can intercept it.